### PR TITLE
fix: consumer address commandhandling to remove relation from backoffice

### PR DESCRIPTION
**Pull request by pgallik**

### CODE REVIEW

File: src/ParcelRegistry.Consumer.Address/BackOfficeConsumer.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public sealed class BackOfficeConsumer : BackgroundService
         private readonly IDbContextFactory<ConsumerAddressContext> _dbContextFactory;
```
Comment (ArneD) [inline src/ParcelRegistry.Consumer.Address/BackOfficeConsumer.cs:19]: not needed anymore

File: paket.dependencies [hunk 2]
```diff
@@ -80,9 +80,10 @@ nuget Be.Vlaanderen.Basisregisters.GrAr.Provenance 17.39.0
 nuget Be.Vlaanderen.Basisregisters.GrAr.Extracts 17.39.0
 nuget Be.Vlaanderen.Basisregisters.GrAr.Oslo 17.39.0
 
-nuget Be.Vlaanderen.Basisregisters.MessageHandling.AwsSqs.Simple 4.6.1
-nuget Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Producer 4.6.1
-nuget Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple 4.6.1
+nuget Be.Vlaanderen.Basisregisters.MessageHandling.AwsSqs.Simple 4.7.0
+nuget Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Producer 4.7.0
+nuget Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple 4.7.0
```
Comment (ArneD) [inline paket.dependencies:85]: Is `Simple` still needed?

File: src/ParcelRegistry.Consumer.Address/paket.template [hunk 3]
```diff
@@ -22,7 +22,6 @@ dependencies
     Be.Vlaanderen.Basisregisters.DataDog.Tracing.SqlStreamStore >= LOCKEDVERSION
     Be.Vlaanderen.Basisregisters.EntityFrameworkCore.EntityTypeConfiguration >= LOCKEDVERSION    
     Be.Vlaanderen.Basisregisters.GrAr.Contracts >= LOCKEDVERSION
-    Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple >= LOCKEDVERSION
```
Comment (ArneD) [inline src/ParcelRegistry.Consumer.Address/paket.template:25]: `Kafka.Consumer.SqlServer` is now needed

File: src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractV2Projections.cs [hunk 4]
```diff
@@ -46,6 +46,9 @@ await context
 
             When<Envelope<ParcelAddressWasAttachedV2>>(async (_, _, _) => await DoNothing());
             When<Envelope<ParcelAddressWasDetachedV2>>(async (_, _, _) => await DoNothing());
+            When<Envelope<ParcelAddressWasDetachedBecauseAddressWasRemoved>>(async (_, _, _) => await DoNothing());
```
Comment (ArneD) [inline src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractV2Projections.cs:49]: bump version?

**Review by ArneD — CHANGES_REQUESTED** [review]

File: src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractV2Projections.cs (hunk 4 above)
Comment (pgallik) [reply src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractV2Projections.cs]: done

**Review by ArneD — APPROVED** [review]

**Pull request merged**